cave/com.raytheon.uf.viz.d2d.nsharp/src/com/raytheon/uf/viz/d2d/nsharp/display/D2DNSharpPaletteWindow.java

```diff
@@ -57,6 +60,11 @@ public class D2DNSharpPaletteWindow extends NsharpPaletteWindow {
     private static final String EDIT_TOOL_CATEGY = "com.raytheon.viz.ui.modalTool.nav";
 
     private AbstractModalTool lastTool = null;
+    
```

Comment:
were these similar changes that we had in v18? or how did you know to make them?